### ISSUE TITLE
Add explanation to emails

### ISSUE THREAD

**Issue by WilliamWDTK** [mysociety/alaveteli#7229]
Often, you'll get a little note at the bottom of an automated email explaining why this has been sent.  There are other issues for specific ones (e.g. track emails), but I think it would be good to consider adding explanations to all/more of them.

**Comment by garethrees** [comment]
I think the first step here would be to write out some specifics of what we'd add to the emails that we think need explanation.

**Comment by HelenWDTK** [comment]
This issue has been automatically closed due to a lack of discussion or resolution for over 12 months.
Should we decide to revisit this issue in the future, it can be reopened.
